### ISSUE TITLE
bias before batch norm

### ISSUE THREAD

**Issue by gunpowder1473** [ANIME305/Anime-GAN#12]
Good evening. Thanks for your work.
I have a question. In you code:
`
        x = spectral_conv2d(x, output_channel, 3, stride=2, is_training=is_training, scope='conv1')
        x = batch_norm(x, is_training, scope='bn2')
        x = tf.nn.leaky_relu(x)
`
It seems that in spectral_conv2d you use the bias, but after batch_norm its effort should be replaced by shift in batch norm. So is it necessary to use a bias?
Thank you!

**Comment by ewrfcas** [comment]
@gunpowder1473 Maybe you are right, i haven't noticed it
